Tabloid/client/src/modules/CategoryManager.js

```diff
@@ -0,0 +1,5 @@
+const baseUrl = `/api/category`;
+
+export const getAllCategories = () => {
```

Comment:
if this route is auth protected, you'll need to get an api token to pass in to this fetch call.

Reply:
Taken care of @heymonicakay